src/test/codegen/issue-45222.rs

```diff
@@ -1,3 +1,5 @@
+// ignore-test LLVM can't prove that these loops terminate.
```

Comment:
This is a regression as well, right? Not great that even trivial examples like these regress...

Reply:
Cf. https://github.com/rust-lang/rust/issues/45222